deploy/helm/values.yaml

```diff
@@ -0,0 +1,13 @@
+replicaCount: 1
+appName: gitlint-bot-production
+appVersion: latest
+image:
+  tag: "latest"
```

Comment:
The image repository is also available for public. You can also put our values of `configmap` and empty `secrets` here for reference.